src/components/appTile/AppTile.js

```diff
@@ -89,6 +91,22 @@ const styles = (theme) => ({
     },
 });
 
+function AppName(props) {
+    const { isDisabled, name, classes, onAppNameClicked, intl } = props;
+    let classname = isDisabled
```

Comment:
Minor suggestion: I would use `const` instead of `let` since these aren't changing.
```suggestion
    const classname = isDisabled
```